packages/rocketchat-api/server/v1/channels.js

```diff
@@ -1,7 +1,7 @@
 //Returns the channel IF found otherwise it will return the failure of why it didn't. Check the `statusCode` property
 function findChannelById({ roomId, checkedArchived = true }) {
 	if (!roomId || !roomId.trim()) {
-		throw new Meteor.Error('error-roomid-param-not-provided', 'The parameter "roomId" is required');
+		throw new Meteor.Error('error-roomid-param-not-provided', 'The parameter "roomId" or "roomName" is required');
```

Comment:
Channels still requires only the `roomId` and not `roomName`, please revert this change and move it to the groups. :+1: